api/util/GenerateTokenScript.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs');
+const readline = require('readline');
+const { google } = require('googleapis');
+
+/**
+ * Prompts user for client ID & secret and writes them to config
+ * file containing auth credentials, then generates a token.json file for GCP
+ * @param {String} configPath the directory path of the config.json file
+ */
+const setAuthCredentials = async (configPath) => {
+  const rawConfig = fs.readFileSync(configPath);
+  console.debug('Welcome to SCE\'s Google Cloud API Token Generator!');
+
+  if(rawConfig) {
+    const clientID = await inputPrompt('Please enter the Client ID: ');
+    const clientSecret = await inputPrompt('Please enter the Client Secret: ');
+
+    const config = JSON.parse(rawConfig);
```

Comment:
you cant modify a `const` variable
```suggestion
    let config = JSON.parse(rawConfig);
```